src/__tests__/pages/api/documentation/file/delete.test.ts

```diff
@@ -13,6 +13,8 @@ jest.mock("../../../../../auth.utils", () => ({
   getSession: () => mockGetSession(),
 }));
 
+jest.mock("@navikt/next-logger");
```

Comment:
Ikke noe galt, bare lurer på hva denne gjør

Reply:
Godt spørsmål! Denne mocken forhindrer støy i outputet til testene når man kjører de i en terminal. Vi tester jo også at ting skal feile "riktig", og for hver feil vi trigger i API-routene våre logger vi feil til next-logger i et format som ser slikt ut:
```
{"level":"error","pid":85394,"hostname":"bekk-mac-2731","msg":"Soknad: 404 error in soknadState - Finner ikke søknad med uuid: 970fa48e-bad2-4665-b05a-000000095704"}
```

Dette er et format som logstash, og dermed Elastic, forstår. Det er veldig kjekt å ha når man skal debugge feil i produksjon, men bare støy når man skal kjøre testene. 😊